src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Colors.values()[random.nextInt(7)].name();
```

Comment:
Please, don't use magic numbers. It's better to retrieve the number of colors directly from Color enumeration:
```suggestion
        Color[] colors = Color.values();
        int index = random.nextInt(colors.length);
        return colors[index].name();
```